src/aws/osml/tile_server/main.py

```diff
@@ -87,16 +85,38 @@ async def lifespan(app: FastAPI) -> AbstractAsyncContextManager[None] | FastAPI:
         They represent your AWS S3 bucket instance,
         your database instance and a queue of requests respectively.
     """
-    viewpoint_worker = ViewpointWorker(viewpoint_request_queue, aws_s3, viewpoint_database, logger)
+    # startup functions before serving requests
+
+    # configure_logging()
+    default_formatter = JsonFormatter(fmt="%(asctime)s %(levelname)s %(message)s", datefmt="%Y-%m-%dT%H:%M:%S")
+    correlation_formatter = JsonFormatter(
+        fmt="%(asctime)s %(levelname)s %(correlation_id)s %(message)s",
+        datefmt="%Y-%m-%dT%H:%M:%S",
+    )
+    worker_filter = ThreadingLocalContextFilter(["correlation_id"])
+    access_filter = CorrelationIdFilter(default_value="-")
+    log_level = ServerConfig.tile_server_log_level
+    uvicorn_logger = logging.getLogger("uvicorn")
+    configure_logger(uvicorn_logger, log_level, log_formatter=default_formatter)
+    worker_logger = uvicorn_logger.getChild("worker")
+    configure_logger(worker_logger, log_level, log_formatter=correlation_formatter, log_filter=worker_filter)
+    uvicorn_error_logger = logging.getLogger("uvicorn.error")
+    configure_logger(uvicorn_error_logger, log_level, log_formatter=default_formatter)
+    uvicorn_access_logger = logging.getLogger("uvicorn.access")
+    configure_logger(uvicorn_access_logger, log_level, log_formatter=correlation_formatter, log_filter=access_filter)
```

Comment:
I think it would be an ideal to move this to `log_tools.py` class

Reply:
I wanted to keep this out of the log_tools.py since it is specific to tile server at the app level instead of a util, but will refactor it into it's own function.